src/axom/spin/internal/linear_bvh/bvh_traverse.hpp

```diff
@@ -36,26 +36,42 @@ inline bool leaf_node(const int32& nodeIdx) { return (nodeIdx < 0); }
  * \param [in] p the primitive in query, e.g., a point, ray, etc.
  * \param [in] B functor that defines the check for the bins
  * \param [in] A functor that defines the leaf action
+ * \param [in] Comp functor used for determining which child node to traverse
+ *  down first if both bins are to be traversed
  *
  * \note The supplied functor `B` is expected to take the following two
  *  arguments:
  *    (1) The supplied primitive, p
  *    (2) a primal::BoundingBox< FloatType, NDIMS > of the BVH bin
  *
+ * \note The supplied functor `Comp` is expected to take the following three
+ *  arguments:
+ *    (1) The left child bounding box
+ *    (2) The right child bounding box
+ *    (3) The primitive being queried
+ *  It should return true if the right child is closer (indicating a swap is
+ *  necessary) and false if the left child is closer.
```

Comment:
Unless I misunderstood, please consider the following for greater clarity.
```suggestion
 *  It should return true if the primitive is closer to the right child bounding
 *  box (indicating a swap is necessary) and false if the primitive is closer to
 *  the left child bounding box.
```